File: 2016/gara2/zeri/att/zeri.pas
function zeri(N: longint; var S: array of char): longint;
begin
    (* Mettete qui il codice della soluzione *)
    zeri := 42;
end;


const
    MAXN = 100000;
var
    N     : longint;
    S     : array[0..MAXN] of char;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, S);

    writeln(fw, zeri(length(S), S));
    close(fr);
    close(fw);
end.

File: ed2/gara2/zeri/att/zeri.c
#include <stdio.h>
#include <string.h>
#include <assert.h>

#define MAXN 100000

int zeri(int N, char S[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


char S[MAXN + 1];

int main() {
    FILE *fr, *fw;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%s", S));

    fprintf(fw, "%d\n", zeri(strlen(S), S));
    fclose(fr);
    fclose(fw);
    return 0;
}
